drivers/include/ph_oem.h

```diff
@@ -0,0 +1,376 @@
+ *
+ * @return zero on successful write, non zero on error
+ */
+int ph_oem_set_interrupt_pin(const ph_oem_t *dev, ph_oem_irq_option_t option);
+
+/**
+ * @brief   Set the LED state of the pH OEM sensor by writing to the
+ *          @ref PH_OEM_REG_LED register
+ *
+ * @param[in] dev       device descriptor
+ * @param[in] state     @ref ph_oem_led_state_t
+ *
+ * @return zero on successful write, non zero on error
+ */
+int ph_oem_set_led_state(const ph_oem_t *dev, ph_oem_led_state_t state);
+
+/**
+ * @brief   Enables/disables the device by putting a high or low signal on
+ *          the GPIO @ref ph_oem_params_t.enable_pin
+ *
+ *          By default the enable pin is defined as @ref GPIO_UNDEF
+ *          if not otherwise defined in the application.
+ *
+ *          The enable pin is a custom PCB feature and not integrated into the
+ *          sensor by default. Your circuit needs an enable pin which controls
+ *          some form of power switch. This function is used to turn off the
+ *          sensor completely, because even in the state @ref PH_OEM_STOP_READINGS
+ *          it still consumes about 3.0mA at 3.3V with the LED turned off.
+ *
+ *          Recommended process:
+ *          1. Enable the device with
+ *             @ref ph_oem_enable_device "ph_oem_enable_device(true)"
+ *
+ *          2. Setup the configurations you need (which are not saved after the
+ *             device was turned off e.g. interrupt)
+ *
+ *          3. Sleep for around 2 minutes for the pH value to stabilize
+ *             (if you need the most accurate result)
+ *
+ *          4. Call @ref ph_oem_set_device_state "ph_oem_set_device_state(PH_OEM_TAKE_READINGS)"
+ *             and wait till the reading was taken (interrupt callback or polling
+ *             if interrupt pin undefined)
+ *
+ *          5. Call @ref ph_oem_read_ph to get the pH value
+ *
+ *          6. Turn the device off again with calling
+ *             @ref ph_oem_enable_device "ph_oem_enable_device(false)"
+ *
+ * @param[in] dev     device descriptor
+ * @param[in] enable  True  = Device enabled <br>
+ *                    False = Device disabled
+ *
+ * @return zero on successful write, non zero on error
+ */
+int ph_oem_enable_device(const ph_oem_t *dev, bool enable);
+
+/**
+ * @brief   Sets the device state (active/hibernate) of the pH OEM sensor by
+ *          writing to the @ref PH_OEM_REG_HIBERNATE register.
+ *          Once the device has been woken up it will continuously take readings
+ *          every 420ms. Waking the device is the only way to take a reading.
+ *          Hibernating the device is the only way to stop taking readings.
+ *
+ * @param[in] dev   device descriptor
+ * @param[in] state @ref ph_oem_device_state_t
+ *
+ * @return zero on successful write, non zero on error
+ */
+int ph_oem_set_device_state(const ph_oem_t *dev, ph_oem_device_state_t state);
+
+/**
+ * @brief   Starts a new reading by setting the device state to
```

Comment:
Maybe you could add that in case of polling it will set the device to STOP mode again.

Reply:
after polling the device and a new reading is available, the device will be set into STOP mode